coral-service/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+  id 'org.springframework.boot' version '2.1.2.RELEASE'
+  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+  id 'java'
+}
+
+group = 'com.linkedin.coral'
+version = '0.0.1'
+sourceCompatibility = '1.8'
+
+repositories {
+  mavenCentral()
+  maven { url 'https://repo.spring.io/milestone' }
+  maven { url "http://conjars.org/repo" }
+}
+
+// Exclusions required to fix issue of multiple bindings on the class path
+configurations.all {
+  exclude module: 'slf4j-log4j12'
+  exclude group: 'javax.servlet'
+}
+
+dependencies {
+  // Spring Boot
+  implementation 'org.springframework.boot:spring-boot-starter-web'
+  testImplementation 'org.springframework.boot:spring-boot-starter-test:2.6.1'
+
+  // Coral
+  implementation 'com.linkedin.coral:coral-common:+'
```

Comment:
Will this grab the last version published as opposed to the current local code? Is there a way to do the latter?